plugins/workspace-backpack/src/backpack_helpers.js

```diff
@@ -231,51 +231,3 @@ export function registerContextMenus(contextMenuOptions, workspace) {
     registerPasteAllBackpack();
   }
 }
-
-/**
- * Converts XML representing a block into text that can be stored in the
- * content array.
- * @param {!Element} xml An XML tree defining the block and any
- *    connected child blocks.
- * @returns {string} Text representing the XML tree, cleaned of all unnecessary
- * attributes.
- */
-export function cleanBlockXML(xml) {
```

Comment:
This method was actually important for making sure we can compare blocks to see if they're equivalent! It removed a lot of extranenous information that we don't care about in the context of the backpack (e.g. the block's old coordinates, its ID, etc).

You'll want to add this functionality to your `blockToJsonString` method.